# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/lutris/no-update-icons.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

diff -urN lutris-0.5.10.1.orig/meson.build lutris-0.5.10.1/meson.build
--- lutris-0.5.10.1.orig/meson.build	2022-06-24 19:40:34.053816224 +0200
+++ lutris-0.5.10.1/meson.build	2022-06-24 19:43:14.686815313 +0200
@@ -98,4 +98,3 @@
   )
 endif
 
-meson.add_install_script('utils/meson_post_install.py')
